src/server/profiles/observation/observation.config.js

```diff
@@ -15,11 +16,14 @@ const scopes = [
 let routes = [
   {
     type: 'get',
-		path: '/dstu2/observation',
+		path: '/:version/observation',
```

Comment:
Are there any differences between the DSTU2 and STU3 versions of the API endpoints?  Presumably the arguments are different, but as long as the endpoints themselves are the same I assume it is safe to combine them.